tests/test_named_ops.py

```diff
@@ -18,8 +18,7 @@
 
 reduction_ops_ut_map = {
     "all": (
-        "test_accuracy_all",
-        "test_accuracy_all_dim",
+        "test_accuracy_all_without_dim",
```

Comment:
We can use pytest mark to group test cases that test a same operator. 
https://docs.pytest.org/en/stable/how-to/mark.html

Reply:
> We can use pytest mark to group test cases that test a same operator. https://docs.pytest.org/en/stable/how-to/mark.html

Thanks, pytest mark will be used at 2nd ed.